application/comit_node/src/swap_protocols/mod.rs

```diff
@@ -15,8 +16,27 @@ pub use self::{
     timestamp::Timestamp,
 };
 
-#[derive(Debug)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, Hash, Copy)]
+pub enum HashFunction {
+    #[serde(rename = "sha-256")]
```

Comment:
:question: @comit-network/rust-devs: Should this be `"SHA-256"` or should the [example](https://github.com/comit-network/RFCs/blob/master/RFC-005-SWAP-Basic-Bitcoin.md#rfc003-swap-request) in RFC003 change?

Reply:
Good catch, thanks.  The standard [1] uses all uppercase, I think we should do likewise.

[1] https://csrc.nist.gov/csrc/media/publications/fips/180/2/archive/2002-08-01/documents/fips180-2.pdf